Packs/MailListener/Integrations/MailListenerV2/MailListenerV2.py

```diff
@@ -574,16 +575,17 @@ def main():     # pragma: no cover
     folder = params.get('folder')
     username = demisto.params().get('credentials').get('identifier')
     password = demisto.params().get('credentials').get('password')
-    verify_ssl = not params.get('insecure', False)
-    tls_connection = params.get('TLS_connection', True)
-    include_raw_body = demisto.params().get('Include_raw_body', False)
+    verify_ssl = not params.get('insecure') or True
+    tls_connection = params.get('TLS_connection') or True
```

Comment:
Lets revert as discussed.
```suggestion
    verify_ssl = not params.get('insecure', False)
    tls_connection = params.get('TLS_connection', True)
```

Reply:
was reverted and discussed